bids/layout/layout.py

```diff
@@ -64,8 +64,57 @@ def get_tr(self, derivatives=False, **entities):
                                .format(entities))
         return all_trs.pop()
 
+class BIDSLayoutVariablesMixin:
+    def get_collections(self, level, types=None, variables=None, merge=False,
```

Comment:
does get_collections() already work using the current state of BIDSLayout? would be great :-)

Reply:
Not quite yet. It's not terribly far though.